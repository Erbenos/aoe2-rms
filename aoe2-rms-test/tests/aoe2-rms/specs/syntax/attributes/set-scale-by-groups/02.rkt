#lang aoe2-rms
; Create 400 tiles worth of lakes, with the number of lakes AND the total number of tiles scaling to map size.  On a small map this will be 4x2.1 = 8 clumps with a total of 400*2.1 = 840 tiles.

<TERRAIN-GENERATION>
(create-terrain 'WATER
                (base-terrain 'GRASS)
                (number-of-tiles 400)
                (number-of-clumps 4)
                (set-scale-by-groups))
