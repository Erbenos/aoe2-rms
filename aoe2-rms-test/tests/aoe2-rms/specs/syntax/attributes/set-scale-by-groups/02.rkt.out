<TERRAIN_GENERATION>
create_terrain WATER {
  base_terrain GRASS
  number_of_tiles 400
  number_of_clumps 4
  set_scale_by_groups
}
